app/labor/article_suggester.rb

```diff
@@ -40,12 +39,11 @@ def suggestions_by_tag(max: 4)
     Article.published.tagged_with(article.tag_list, any: true).
       where.not(id: article.id).
       order("hotness_score DESC").
-      includes(user: [:pro_membership]).
       offset(rand(0..offsets[0])).
       first(max)
   end
 
   def offsets
-    Rails.env.production? ? [10, 120] : [0, 0]
+    Rails.env.production? ? [150, 1000] : [0, 0]
```

Comment:
it'd be great if we could document the reason behind these numbers, as you said in the PR, this way the thought process will stay in the code

Reply:
Also, is there a reason why we don't just return the two numbers and we differentiate between production and everything else? Both in development and testing 150 and 1000 items are very unlikely to be in the DB (which wouldn't matter in development anyway) and in the worst case scenario this only slows down a few tests.

Reply:
Do we do a lot of ifs check whether it's prod or not? Wouldn't this make more sense in the site configuration?